### DISCUSSION TITLE
Find words in bookmarks list

### DISCUSSION THREAD

**Discussion by NurudinImsirovic** [sumatrapdfreader/sumatrapdf#4137] [Q&A]
I'm reading a documentation file and need to find all bookmarks matching a specific word.
Unfortunately I'm unable to find an option for doing this.

![image](https://github.com/sumatrapdfreader/sumatrapdf/assets/102489121/9e8d9e88-42d7-4a59-93fe-bbf032784778)

**Comment by GitHubRulesOK** [comment]
It has been a open issue for a while 
see https://github.com/sumatrapdfreader/sumatrapdf/issues/943
